INSTALL-OLD.md

```diff
@@ -0,0 +1,212 @@
+**This installation guide is deprecated and might be out
+of date! It is recommended that you deploy using
+[Docker](https://github.com/rr-/szurubooru/blob/master/DOCKER.md)
```

Comment:
Since you've renamed `DOCKER.md` to `INSTALL.md`, this link should be adjusted accordingly.